website/blog/2019-08-21-announcing-riff-0-4-0.md

```diff
@@ -0,0 +1,124 @@
+
+A lot has changed, buckle up.
+
+<!--truncate-->
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/cli/releases/tag/v0.4.0) on GitHub. Please follow one of the [getting started](/docs/v0.4/getting-started) guides, to create a new cluster on GKE, Minikube, Docker Desktop for Mac, or Docker Desktop for Windows.
+
+## Notable changes
+
+### Knative and Istio
+
+riff no longer has a hard dependency on Knative and Istio. Knative integration is provided by riff's [Knative runtime](#knative-runtime), which is available in clusters with Knative Serving installed. Istio is only required for the Knative runtime.
+
+### builds
+
+riff Builds now encompass more than just functions with the addition of application and container builds. Each build resource reports that latest image that a [runtime](#runtimes) uses to deploy the built workload. The built image is always reported as a digested image to make the detection and rollout of changes more deterministic.
+
+Builds are transitioning from a one-off tasks to a continuous stream of images. riff 0.4 sets up the API to enable ongoing updated builds, but does not yet hook into sources to trigger rebuilding.
+
+#### function builds
+
+A Function uses function buildpacks and invokers to convert function source code into a runnable container. The same invokers and buildpacks provided by riff 0.3 are available. Unlike riff 0.3, the Function is only responsible for building a container image. The deployment of the container is managed by the [runtime](#runtimes).
+
+#### application builds
+
+In addition to functions, riff can now build Applications using the [Cloud Foundry, Cloud Native Buildpack builder](https://hub.docker.com/r/cloudfoundry/cnb).
+
+#### container builds
+
+A Container is a build-less build. It exists to resolve an existing container image providing the same interface for other resources to consume as Application and Function. The image is resolved to a digest and checked for an updated image every 5 minutes.
+
+### runtimes
+
+riff Runtimes provide different deployment experiences for a workload. A given riff cluster may not support each runtime.
+
+#### Core runtime
+
+The Core runtime is a streamlined runtime that allows operators to provide their own scalers, observability and ingress. For each Deployer, the runtime creates a Kubernetes Deployment and a Service that targets the deployment. The deployment is automatically updated for new build images creating a new replicaset based on the rollout strategy.
+
+The workload is accessible from within the cluster by default, but must be explicitly exposed externally. A single replica is run by default, but the deployment can be targeted by a HorizontalPodAutoscaler or any other scaler that supports the `/scale` subresource on deployment.
```

Comment:
I would maybe move the part about operators bringing "their own scalers, observability and ingress" to follow the above paragraph.